TheMovieDb/Models/MoviesList.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  MovieList.swift
+//  TheMovieDb
+//
+//  Created by Alex on 11/4/21.
+//
+
+import Foundation
+
+struct MovieItem: Decodable {
+    let id: String
+    let name: String
+
+    enum CodingKeys: String, CodingKey {
+        case id
+        case name
+        case url
+    }
+    
+    init(from decoder: Decoder) throws {
```

Comment:
I still believe that we could use a simpler approach to decoding; since we are using the same names that come from the service as property names, it should not be necessary to add `CodingKeys` and the `init(from decoder)` method.

Reply:
You are right, this can be very simple.